pkgs/development/python-modules/vharfbuzz/default.nix

```diff
@@ -8,11 +8,11 @@
 
 buildPythonPackage rec {
   pname = "vharfbuzz";
-  version = "0.2.0";
+  version = "0.1.4";
```

Comment:
I assume this downgrade decision might need to be revisited now - maybe upstream improved the situation. If not, I'd squash this commit and it's parent into a single commit like this:

```
python3Packages.vharfbuzz0_1_4: init at 0.1.4
```

Then, put the information answering the question "why isn't the latest and greatest version of this project packaged?", that is available in the commit message, near the version string.

Reply:
Actually it was nixpkgs not upstream that was to blame here :-) our uharfbuzz was too old, and I was planning to send a PR at some point to upgrade it... of course in the meantime it has already been upgraded, so this indeed no longer an issue.